.devcontainer/Dockerfile

```diff
@@ -6,15 +6,15 @@ FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Terraform, providers and tflint versions
-ARG TERRAFORM_VERSION=1.0.3
-ARG AZURERM_VERSION=2.69.0
+ARG TERRAFORM_VERSION=1.0.0
```

Comment:
The TF version is pinned to `= 1.04` in main.tf and tier3.tf

Reply:
Fixed!